insteon_mqtt/handler/StandardCmd.py

```diff
@@ -86,8 +86,22 @@ def msg_received(self, protocol, msg):
             # If this message matches our address and command, it's probably
             # the ACK we're expecting.
             if msg.from_addr == self.addr and msg.cmd1 == self.cmd:
-                # Run the callback - it's up to the callback to check if this
-                # is really the ACK or not.
+                if msg.flags.type == Msg.Flags.Type.DIRECT_NAK:
+                    if msg.cmd2 == msg.NakType.PRE_NAK:
```

Comment:
If it would be useful, I had coded up some changes that would allow PRE_NAK to be only processed as such for i2cs devices.  It basically involves passing the device's self.db.engine value to the StandardCmd constructor everywhere these messages are used, though, which I wasn't thrilled about.  Unfortunately, I didn't see an easy way around it, since the handlers don't otherwise know anything about the device they're handling for.  Let me know if you'd like me to adapt my changes to fit in with your recent commits.

Alternatively, it probably wouldn't be all that problematic as is.  Maybe an on command to level 0xFC that gets a NAK from an old device that just echos cmd2 back will look like a Pre-NAK instead, now.

Reply:
Yeah, I don't think this is necessary.  The message is still a NAK, which in this handler never gets passed on.  Worst case is that an old i1 device waits 2x as long to time out with some weird error logging.